### PR TITLE
Fix Travis CI ignoring non-last command errors in each step

### LINKED ISSUE #6075: Travis test steps would ignore failures in non-last commands
First reported in #6068.
If a step of test consists of multiple commands, errors caused in non-last commands are being ignored.

Example:
https://travis-ci.org/chainer/chainer/jobs/483290510

Cause:
https://github.com/chainer/chainer/pull/6068#issuecomment-456829783

### PR DESCRIPTION
**Pull request by niboshi**
Fixes #6075

### CODE REVIEW

File: scripts/ci/travis/steps.sh [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+        "$REPO_DIR"/chainerx_cc/examples
+    )
+
+    # Check all targets exist
+    for f in "${check_targets[@]}"; do test -e "$f"; done
+
+    flake8 --version
+    flake8 "${check_targets[@]}"
+
+    false  # TODO: remove me
```
Comment (niboshi) [inline scripts/ci/travis/steps.sh]: This command emulates a test failure.
After review, I'll remove this line and confirm CI pass in turn.

**Comment by niboshi** [comment]
@hvy 
That won't fix the problem.
Please read this: https://github.com/chainer/chainer/pull/6068#issuecomment-456829783

In a compound `func1 || cmd`, where a non-last command in `func` fails, `func` will not return non-zero code.

**Comment by hvy** [comment]
LGTM!

File: scripts/ci/travis/steps.sh (hunk 1 above)
Comment (hvy) [reply scripts/ci/travis/steps.sh]: I think it's okay to merge, could you revert this temporary modification?

**Comment by niboshi** [comment]
@hvy Removed the temporary command.

**Comment by hvy** [comment]
jenkins, test this please.

**Comment by hvy** [comment]
CI failures seem unrelated.

**Pull request merged**